src/createNumbersGenerator.js

```diff
@@ -25,8 +25,11 @@
  *
  * @return {number}
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(numbers = [], min = -Infinity, max = Infinity) {
+  let countIndex = 0;
+  const numbersArray = numbers.filter((item) => item >= min && item <= max);
```

Comment:
```suggestion
  const numbersArray = numbers.filter(number => number >= min && number <= max);
```

Reply:
Thanks! It's really hard to relearn not to write like this.